src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -410,21 +434,59 @@ public void updateDesiredLocation(DesiredLocation location) {
    */
   public Pose2d findOTFPoseFromPathLocation() {
     switch (this.desiredLocation) {
-        // reef 0 and 1 will have the same path
-        // NOTE: use PathPlannerPath.getStartingHolonomicPose to find pose for reef lineup if wanted
       case Reef0:
         return DriverStation.getAlliance().orElse(Alliance.Blue) == Alliance.Red
-            ? JsonConstants.redFieldLocations.reef0
-            : new Pose2d();
+            ? new Pose2d(
```

Comment:
thank you so much for actually filling out all these fields, this is extremely based